wiki/Contests/A_Labour_of_Love_Contest/en.md

```diff
@@ -14,13 +14,14 @@ Each submission is prepared by a team. Teams can submit their name and members i
+| Bowkutoly Prizmcore | ![][flag_TH] [Raytoly](https://osu.ppy.sh/users/8121109) | ![][flag_TH] [Raytoly](https://osu.ppy.sh/users/8121109), ![][flag_GB] [Agent Encore](https://osu.ppy.sh/users/18121148), ![][flag_ID] [gokugohan12468](https://osu.ppy.sh/users/2013571), ![][flag_US] [PrizmClicks](https://osu.ppy.sh/users/15420104) | ![][flag_GB] [Agent Encore](https://osu.ppy.sh/users/18121148), ![][flag_US] [BowLL](https://osu.ppy.sh/users/10198015), ![][flag_US] [PrizmClicks](https://osu.ppy.sh/users/15420104) | 
 | Cranked Love | ![][flag_UA] [Rentra](https://osu.ppy.sh/users/8873826) | ![][flag_US] [Codekid](https://osu.ppy.sh/users/18749383) | ![][flag_UA] [Rentra](https://osu.ppy.sh/users/8873826), ![][flag_US] [Codekid](https://osu.ppy.sh/users/18749383) | 
 | Daiten | ![][flag_CL] [daikiu](https://osu.ppy.sh/users/4232665), ![][flag_ID] [Shouten](https://osu.ppy.sh/users/16593295) | ![][flag_CL] [daikiu](https://osu.ppy.sh/users/4232665), ![][flag_ID] [Shouten](https://osu.ppy.sh/users/16593295) | ![][flag_CL] [daikiu](https://osu.ppy.sh/users/4232665), ![][flag_ID] [Shouten](https://osu.ppy.sh/users/16593295) | 
 | Daniel Gaming | ![][flag_US] [Sooph](https://osu.ppy.sh/users/12001243) | ![][flag_ID] [Tocorn](https://osu.ppy.sh/users/9564072), ![][flag_US] [\_qqpp\_](https://osu.ppy.sh/users/13181574) | ![][flag_US] [Malishiosu](https://osu.ppy.sh/users/12521528) | 
 | Dolbit Normalno | ![][flag_RU] [DestoppeD](https://osu.ppy.sh/users/19611631) | ![][flag_RU] [NeilPerry](https://osu.ppy.sh/users/841391), ![][flag_CA] [Sing](https://osu.ppy.sh/users/3795679), ![][flag_RU] [Mirash](https://osu.ppy.sh/users/2841009) | ![][flag_RU] [Tommy Phelps](https://osu.ppy.sh/users/10974581) | 
 | extremely valid | ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471), ![][flag_FR] [Shinwoir](https://osu.ppy.sh/users/8984574) | ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515), ![][flag_GB] [DeviousPanda](https://osu.ppy.sh/users/4966334), ![][flag_FR] [Realazy](https://osu.ppy.sh/users/918297) | ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515), ![][flag_FR] [Realazy](https://osu.ppy.sh/users/918297) | 
 | humilde | ![][flag_AR] [pm04034](https://osu.ppy.sh/users/12704335) | ![][flag_AR] [Megafan](https://osu.ppy.sh/users/6632605), ![][flag_TH] [ohm002](https://osu.ppy.sh/users/4468239) | ![][flag_TH] [ohm002](https://osu.ppy.sh/users/4468239) | 
+| Incoherent Sound | ![][flag_MY] [takehirotei](https://osu.ppy.sh/users/11793794) | ![][flag_MY] [Kardshark](https://osu.ppy.sh/users/4724315), ![][flag_MY] [[MY]xArief](https://osu.ppy.sh/users/12694468), ![][flag_MY] [[-Chocola-]](https://osu.ppy.sh/users/6781232) | ![][flag_MY] [GhostFY](https://osu.ppy.sh/users/7798305) | 
```

Comment:
```suggestion
| Incoherent Sound | ![][flag_MY] [takehirotei](https://osu.ppy.sh/users/11793794) | ![][flag_MY] [Kardshark](https://osu.ppy.sh/users/4724315), ![][flag_MY] [\[MY\]xArief](https://osu.ppy.sh/users/12694468), ![][flag_MY] [\[-Chocola-\]](https://osu.ppy.sh/users/6781232) | ![][flag_MY] [GhostFY](https://osu.ppy.sh/users/7798305) | 
```
escaping

Reply:
jsyk this isn't necessary, you only need to escape things if they break gfm or commonmark